controllers/compaction/metrics.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2023 SAP SE or an SAP affiliate company
```

Comment:
Does it also make sense to introduce another metric `metricJobsCurrentCount` which keeps track of the number of compaction jobs being run currently by druid, so that certain scheduling decisions may be taken based on number of available machines to schedule these compaction jobs.

Reply:
Done